src/aat/resources/features/F-1001/F-1001.feature

```diff
@@ -0,0 +1,156 @@
+@F-1001
+Feature: F-1001: Get case document metadata for a given caseId and documentId
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-1010
+  Scenario: must receive a document metadata and list of permissions for a valid Citizen user
+    Given a user with [an active Citizen profile in CCD with access to a document]
```

Comment:
it reads like there is a user and a document in the context and the user has access to the doc. However, at this stage of the scenario, the document is not yet on CCD.